src/pages/Details.vue

```diff
@@ -76,7 +77,7 @@
                 <div class="row">
                     <div class="col-md-8">
                         <HeartbeatBar :monitor-id="monitor.id" />
-                        <span class="word">{{ $t("checkEverySecond", [ monitor.interval ]) }}</span>
+                        <div><span v-if="monitor.active" class="mr-2">Next in {{ formattedTime }}&nbsp;</span><span class="word">({{ $t("checkEverySecond", [ monitor.interval ]) }})</span></div>
```

Comment:
hey could u help in adding languge thing? is there a file where i can add in this text so tht it converts to other languages automatically?

Reply:
Should have been more specific:
We use vue-i18n to translate the frontend.
Relevant docs for this PR (depending on which usage we end up with)
- https://vue-i18n.intlify.dev/guide/essentials/syntax.html
- https://vue-i18n.intlify.dev/guide/advanced/component
- https://vue-i18n.intlify.dev/guide/essentials/pluralization.html

You need to add all added translation keys to `en.json` (and no other language to prevent merge conflicts with weblate)